cloud-config/azure/README.md

```diff
@@ -0,0 +1,89 @@
+https://portal.azure.com/#blade/HubsExtension/Resources/resourceType/Microsoft.Compute%2FVirtualMachines
+
+**To use this setup, you must able to link a domain name to EC2 machine's IP address. If you don't own a domain, services such as [FreeDNS](https://freedns.afraid.org) offer free sub-domains under a range of domains.**
+
+## Prerequisites
+
+**Warning**: Make sure at all time that you have selected the availability zone where you want to perform your actions. You can choose the availability zone using the dropdown menu at the top-right corner of the AWS console website.
+
+### Create a VPC
+
+- Go to the [Azure Virtual Machines Dashboard](https://portal.azure.com/?l=en.en-us#blade/HubsExtension/Resources/resourceType/Microsoft.Compute%2FVirtualMachines)
+  
+  Click on `Add`.
+
+- Review the Project Details section.
+
+  Ensure that a `Subscription` and a `Resource group` is selected.
+  
+- Review the Instance Details section.
+
+  Give your VM a name in the `Virtual Machine name` field.
+  
+  Select a region.
+  
+  Select `Ubuntu Server 18.04 LTS` from the `Image` dropdown.
+  
+  Change the size of the Vm according to your Aggregate needs.
+  
+- Review the Administrator Account section.
+
+  Fill in the credentials of the user account that will be created during the creation of the VM. You will use this user account to complete the Aggregate installation later.
+  
+- Review the Inbound Port Rules section.
+
+  Select the `Allow selected ports` option in the `Public inbound ports` field.
+  
+  Select `HTTP`, `HTTPS`, and `SSH` in the `Select inbound ports` field.
+  
+- Review the Network configuration.
```

Comment:
The more natural thing is to click to at the bottom to get to the disks, so point out the Networking tab at top (or tell em to click through disks).